youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -367,6 +367,10 @@ def run(self, info):
         if info.get('album') is not None:
             metadata['album'] = info['album']
 
+        for m in self._downloader.params.get('custommeta'):
+            key,val = m.split('=',1)
+            metadata[key] = val.replace('\\n', '\n') % info
```

Comment:
Missing fields should be raplaced with `NA` similarly to [output template](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/YoutubeDL.py#L568).